model/sql-query-model.js

```diff
@@ -63,4 +65,6 @@ exports.queryPostNewComment = async function(article_id, newComment)
         RETURNING *;`, newComment.body, newComment.username, article_id));
     
         return postedComment.rows[0];
-}
\ No newline at end of file
+}
+
+//No new tests. Visual confirmation has been undertaken.
```

Comment:
As this is new functionality to the path and it has new cases for errors, it needs the following tests
  - [ ] Status 200, default sort & order: `created_at`, `desc`
  - [ ] Status 200, accepts `sort_by` query, e.g. `?sort_by=votes`
  - [ ] Status 200, accepts `order` query, e.g. `?order=desc`
  - [ ] Status 200, accepts `topic` query, e.g. `?topic=coding`
  - [ ] Status 400. invalid `sort_by` query, e.g. `?sort_by=bananas`
  - [ ] Status 400. invalid `order` query, e.g. `?order=bananas`
  - [ ] Status 404. non-existent `topic` query, e.g. `?topic=bananas`
  - [ ] Status 200. valid `topic` query, but has no articles responds with an empty array of articles, e.g. `?topic=paper`